source/Introduction/About-SpaceROS.rst

```diff
@@ -1,12 +1,96 @@
+Manipulators, like the Canadarm, that can deploy payloads; humanoid robots, like Robonaut 5, that can perform tasks that require manual dexterity in the hands and fingers; mobility systems, like the VIPER rover, that can traverse planetary surfaces; robotic spacecraft; and robotic landers.
+In the future, robots will be used for all sorts of tasks, like mining and excavation, maintenance and repair, in-space manufacturing, and site preparation.
+
+.. image:: /images/canadarm.jpg
+
+.. rst-class:: image-subtitle
+
+    The Canadarm manipulator
+
+Increase in Software Complexity
+###############################
+
+With these various kinds of applications comes the corresponding increase in the overall complexity of the systems, especially the associated software.
+The amount of software on the robotic systems is increasing at a steep rate.
+Consider the joystick controlled Lunokohd rover of 1970:
+
+.. image:: /images/lunokhod.jpg
+
+.. rst-class:: image-subtitle
+
+    The Lunokhod rover, circa 1970
+
+The amount of code on that rover was certainly minimal when compared to the 2 million+ lines of code required for the Mars rover in 2020.
+Today, writing custom software for each mission is no longer a viable option as this approach typically results in:
+
+* Higher development costs
+* Minimal code reuse
+* Time-consuming and costly qualification to meet required safety standards
+
+The result is that there is an increasing demand for software re-use in space systems.
+
+..
+    In fact, the space community has been moving in the direction of re-usable software for some time, with flight software frameworks like FPrime and cFS, as well as mission control software, like Yamcs and OpenMCT.
+    There are also smaller open source projects, like AprilTag, that have been used on space missions.
+    In the picture here you can see the tags on the Perserverence rover, for example.
+    These tags can be used as a point of reference or can be scanned to provide information, kind of like a QR code.
+    The trend towards using open source software and reuse, in general, for space systems is clear.
+
+.. image:: /images/ros_logo.png
+    :width: 125
+
+Meanwhile, back here on Earth, the broader robotics community has standardized on ROS, an open source platform for building robotics applications.
+ROS includes communications middleware, drivers that interface to hardware, state-of-the-art algorithms, and a wide range of developer tools.
+Implementing a robot for space applications can be greatly simplified by incorporating a robotics framework like ROS.
+For example, ROS packages like `MoveIt2 <https://github.com/ros-planning/moveit2>`_ can plan and execute motion for manipulators, like the Canadarm or the Robonaut with its hands and fingers.
+Other packages like `Navigation2 <https://github.com/ros-planning/navigation2>`_ provide the means for robots to perform dynamic path planning and navigation from point A to point B, while avoiding obstacles.
```

Comment:
Would it make sense for the MoveIt and Navigation links to point to https://moveit.ros.org and https://navigation.ros.org/ respectively rather than the project repositories?
```suggestion
For example, ROS packages like `MoveIt2 <https://moveit.ros.org>`_ can plan and execute motion for manipulators, like the Canadarm or the Robonaut with its hands and fingers.
Other packages like `Navigation2 <https://navigation.ros.org/>`_ provide the means for robots to perform dynamic path planning and navigation from point A to point B, while avoiding obstacles.
```

Reply:
Yes, good idea. Updated.